packages/most-buffer/test/test-buffer.js

```diff
@@ -4,17 +4,33 @@ const most = require('most');
 
 describe('buffer', function() {
   it('can group stream events 10 by 10', function() {
+    let count = 0;
     return most.iterate(x => x + 1, 0)
-      .take(60)
+      .take(25)
       .thru(buffer(10))
+      .tap(() => { count += 1; })
```

Comment:
Use `reduce` method to consume the stream and count the buffers that has been emitted.